### PR TITLE
Port Driver Station Log Viewer from Screensteps

### LINKED ISSUE #120: DriverStation log viewer needs to be ported
https://wpilib.screenstepslive.com/s/currentCS/m/driver_station/l/144980-driver-station-log-file-viewer needs a port. There's a pull-request open but I believe that's stale.

### PR DESCRIPTION
**Pull request by DylanB5402**
Closes #120

### CODE REVIEW

**Comment by yellowjaguar5** [comment]
Whoops, I think I might have already done this... https://github.com/wpilibsuite/frc-docs/pull/131/files#diff-0f71f7ccb3ee2aaef1f87719295047d2

**Comment by Daltz333** [comment]
I'll prefer this PR since it was assigned to him in the issue beforehand.

**Comment by AustinShalit** [comment]
@DylanB5402 You need to update the branch from master.  You can do this by clicking the "Update branch" button on GitHub and then pulling the changes into your local copy (`git pull`).

File: source/docs/software/driverstation/driver-station-log-viewer.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Radio reboot
+~~~~~~~~~~~~
+
+.. image:: images/driver-station-log-viewer/radioreboot.png
+
+A reboot of the robot radio is typically characterized by a loss of connection to the radio for ~25-30 seconds. In this example, the radio briefly lost power at 3:22:44, causing it to start rebooting. The event at 3:22:45 indicates that the ping to the radio failed. At 3:23:11, the DS regains communication with the roboRIO and the roboRIO indicates it gas been up for 1272.775 seconds, ruling out a roboRIO reboot. Note that the network switch on the radio comes back up very quickly so a momentary power loss may not result in a "radio lost"/"radio seen" event pair. A longer disturbance may result in radio events being logged by the DS. In that case, the distinguishing factor which points towards a radio reboot is the ping status of the radio from the DS. If the radio resets, the radio will be unreachable. If the issue is a cabling or connection issue on the robot, the radio ping should remain "GOOD".
```
Comment (AustinShalit) [inline source/docs/software/driverstation/driver-station-log-viewer.rst]: gas -> has and a radio reboot is ~40 seconds.
```suggestion
A reboot of the robot radio is typically characterized by a loss of connection to the radio for ~40-45 seconds. In this example, the radio briefly lost power at 3:22:44, causing it to start rebooting. The event at 3:22:45 indicates that the ping to the radio failed. At 3:23:11, the DS regains communication with the roboRIO and the roboRIO indicates it has been up for 1272.775 seconds, ruling out a roboRIO reboot. Note that the network switch on the radio comes back up very quickly so a momentary power loss may not result in a "radio lost"/"radio seen" event pair. A longer disturbance may result in radio events being logged by the DS. In that case, the distinguishing factor which points towards a radio reboot is the ping status of the radio from the DS. If the radio resets, the radio will be unreachable. If the issue is a cabling or connection issue on the robot, the radio ping should remain "GOOD".
```

**Pull request merged**